src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
```

Comment:
I think we should pull these colors from our [web-commons-ui-assets](https://gitlab.com/nielsen-media/maf/web-commons-ui-assets) so if anything will change there, we will get the change out of the box.